src/components/Hero.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Typography, Grid, useMediaQuery } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import HeroImage from '../../public/hero-img.png';
```

Comment:
This image should not be hardcoded, as far as we agreed on. Or is this WIP? 
Because this image link should come from the API. Also as far as I remember it can contain mobile, desktop, and tablet versions.

Reply:
Regarding mobile images, for some components we 'switch' to square image ratios on small mobile (this should be noted in the GitHub task if so). But for hero component, the image ratio stays the same, so the same hero image can be used across all devices.

Reply:
At our frontend kickoff meeting last week, we were discussing how it might be better to save the images in code for now, since it's not great to have many requests sent to my personal google drive folder where the images are saved temporarily. But maybe @stepsen89 can elaborate? :)

Reply:
@dricazenck @lilyndk  I'd wait for @stepsen89 's response to see if I need to make any changes!